packages/esm-service-queues-app/src/queue-table/queue-table-by-status.component.tsx

```diff
@@ -50,15 +50,8 @@ const QueueTableByStatus: React.FC<QueueTableByStatusProps> = ({
   const { queueEntries, isLoading } = useQueueEntries({ queue: selectedQueue.uuid });
   const allowedStatuses = selectedQueue.allowedStatuses;
 
-  const currentStatusUuid = selectedStatus?.uuid;
-
-  const currentStatusIndex = Math.max(0, allowedStatuses?.findIndex((s) => s.uuid == currentStatusUuid));
-
-  const queueEntriesForCurrentStatus = queueEntries?.filter(
-    (entry) => !currentStatusUuid || entry.status.uuid == currentStatusUuid,
-  );
-
-  const tableConfig = configByStatus?.get(currentStatusUuid) ?? defaultQueueTableConfig;
+  const currentStatusUuid = selectedStatus?.uuid ?? allowedStatuses?.[0]?.uuid;
```

Comment:
What are the implications of defaulting here to the first entry in allowed statuses?  Is that really what we want?

Reply:
Currently, without a status uuid specified in the URL, the tab selection defaults to the first status but shows entries from all statuses. We should default it to something. An alternative would be defaulting to "all statuses", but with how the queue table is designed right now we can't assume that the "all statuses" tab is available. Another alternative would be to only support URLs with both queue and status uuid specified.